### ISSUE TITLE
Add Impeller to the Swift Package Catalog

### ISSUE THREAD

**Issue by david-coyle-sjc** [mentalfaculty/impeller#14]
Found here:
https://packagecatalog.com

I'd do it myself, but the entry ought best be owned and maintained by the package's owner as well.

Feel free to wait until Impeller is more stable, OTOH, this will give you more visibility.

**Comment by drewmccormack** [comment]
Good idea. I will indeed wait a little. Would like to at least have atomic persistence (eg plist), and probably SQLite too, before doing that. But in a month or so I think I will likely be ready.

**Comment by david-coyle-sjc** [comment]
I have shared this project with a former WO engineer, we may get some traction, I see great potential.
